src/common/java/net/minecraftforge/gradle/common/legacy/LegacyExtension.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.gradle.common.legacy;
+
+import groovy.lang.GroovyObjectSupport;
+import net.minecraftforge.gradle.common.tasks.ExtractZip;
+import net.minecraftforge.gradle.common.util.MinecraftExtension;
+import net.minecraftforge.srgutils.MinecraftVersion;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Dependency;
+import org.gradle.api.file.FileCollection;
+import org.gradle.api.provider.Property;
+import org.gradle.api.provider.Provider;
+
+import java.io.File;
+
+
+/**
+ * Provides an extension block named "legacy".
+ *
+ * <code>
+ * legacy {
+ *     flag = set;
+ * }</code>
```

Comment:
Abstract documentation for the purpose of having documentation... Why exactly does this exist?

Reply:
Some history: this was written before everything else in the LegacyExtension, as a guide and reminder so that i knew what to search for when building this.

It is not needed any more, so it has been removed.